src/App.js

```diff
@@ -9,12 +9,11 @@ import "./styles/styles.css"
 
 export default function App() {
   const [contacts, setContacts] = useState([])
-  const [sensor,setSensor] = useState(false)
   
   useEffect(() => contacts && fetch("http://localhost:4000/contacts")
```

Comment:
Awesome Paul! very nice use of conditional rendering in useEffect. Personally wasnt aware it could be used inside useEffect!